tests/integration/vlan_test.py

```diff
@@ -45,6 +45,40 @@ def test_add_and_remove_two_vlans_on_same_iface(eth1_up):
     assert not current_state[INTERFACES]
 
 
+def test_rollback_for_vlans(eth1_up):
+    current_state = statelib.show_only([])
+    desired_state = {
```

Comment:
Since this is the same state as in ` two_vlans_on_eth1()` it makes sense to move them to a constant and use it in both methods IMHO.